benchmarks/index.js

```diff
@@ -0,0 +1,49 @@
+import {text, buffer, arrayBuffer} from 'node:stream/consumers';
+import getStream, {getStreamAsBuffer, getStreamAsArrayBuffer, getStreamAsArray} from '../index.js';
+import {createFixture, deleteFixture, FIXTURE_HUMAN_SIZE} from './fixture.js';
+import {measureTask} from './measure.js';
+import {createNodeStreamBinary, createNodeStreamText, createWebStreamBinary, createWebStreamText} from './stream.js';
+
+await createFixture();
+
```

Comment:
Maybe use a helper method? Something like this.

```js
const log = (name, promise) => {
  console.log(`- \`${name}()\`: ${await promise}ms`);
};

Reply:
Thanks, this is fixed in 8f0ebe7. This looks much better now, my original code was quite messy. :)